v2/cn/download.md.tt2

```diff
@@ -4,6 +4,11 @@
     @created       2011-06-21 04:25 GMT
 --->
 
+对于一些常见的 Linux 发行版本，OpenResty 提供 [官方预编译包](linux-packages.html)。确保你首先用这种方式来安装。
+
+你可以下载下面最新的版本。然后按照 [安装](installation.html)页面的说明来 build 和安装。
```

Comment:
This is not quite right because we also ensure that the official Linux packages are also at the latest version.

I've adjusted the English version a bit:

"You can download the source release tarballs below."